addons/website/static/src/js/menu/edit.js

```diff
@@ -132,11 +132,10 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
      * @private
      */
     _addEditorMessages: function () {
-        var $target = this._targetForEdition();
-        this.$editorMessageElements = $target
-            .find('.oe_structure.oe_empty, [data-oe-type="html"]')
-            .not('[data-editor-message]')
-            .attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
+        const $target = this._targetForEdition();
+        this.$editorMessageElements = $target.find('.oe_structure.oe_empty, [data-oe-type="html"]');
+        this.$editorMessageElements.not('[data-editor-message]').attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
+        this.$editorMessageElements.attr('contentEditable', !this.$editorMessageElements.is(':empty'));
```

Comment:
How does this work? Once I drop a snippet in the page, what code is making the page editable again? :thinking:

Reply:
The method is executed again after adding a snippet in the page.